pyvista/plotting/charts.py

```diff
@@ -4742,6 +4742,11 @@ def _renderer(self):
         """Return the weakly dereferenced renderer, maybe None."""
         return self.__renderer()
 
+    @property
+    def charts(self):  # numpydoc ignore=RT01
```

Comment:
This shouldn't be needed as far as I know. The `Charts` object implements the `__getitem__`, `__len__` and `__iter__` methods, so it should be usable as any other list. Can you provide an example on why access to the explicit list would be needed?

Reply:
I missed that!